META-INF/persistence.xml

```diff
@@ -2,96 +2,96 @@
 <persistence xmlns="http://java.sun.com/xml/ns/persistence" version="2.0">
```

Comment:
According to the documentation this file is not necessary since you are using the latest version of Spring.
"Usually, JPA defines a persistence unit through the META-INF/persistence.xml file. Starting with Spring 3.1, the persistence.xml is no longer necessary. "

Reply:
Im not sure if this one was created by default by IntelliJ IDE, i will be looking for more info about this.